src/napari_imagej/widgets/menu.py

```diff
@@ -227,8 +228,13 @@ def get_chosen_layer(self) -> None:
                 self._add_image(image)
 
     def get_active_layer(self) -> None:
-        # Choose the active DatasetView
-        view = ij().get("net.imagej.display.ImageDisplayService").getActiveDatasetView()
+        # HACK: Sync ImagePlus before transferring
+        if ij().legacy and ij().legacy.isActive():
+            ij().py.sync_image(ij().WindowManager.getCurrentImage())
```

Comment:
This will fail if `getCurrentImage()` returns None. I suggest splitting this code into two lines, with a null check on the imp.

Reply:
Thanks, fixed in e0b3064